evalml/tests/component_tests/test_lsa.py

```diff
@@ -97,15 +65,16 @@ def test_invalid_text_column():
             np.nan,
             None,
             'I\'m happy again!!! lalalalalalalalalalala']})
-    lsa = LSA(text_columns=['col_1'])
+    X = infer_feature_types(X, {'col_1': 'NaturalLanguage'})
```

Comment:
Does this mean that if a user had this dataframe `X` and didn't run `infer_feature_types`, then ww wouldn't pick up on the column being of type `NaturalLanguage`?

Reply:
I believe it does, but in case the heuristic for determining types changes, adding this to assert that we want this to be natural language :d